### DISCUSSION TITLE
Why does my x86 build build x64 when i have a custom triplet in manifest mode?

### DISCUSSION THREAD

**Discussion by dretax** [microsoft/vcpkg#34738] [General]
Beginning to install....  x64-windows
Executing...  x64-windows "d:\Projects\Kernel\vcpkg\vcpkg.exe" install --x-manifest-root=".....\buildResources" --x-wait-for-lock --triplet "x64-windows" --x-install-root "......\buildResources\x64-windows\"
Beginning to install....  x86-windows
Executing...  x86-windows ".....\vcpkg\vcpkg.exe" install --x-manifest-root=".....\buildResources" --x-wait-for-lock --triplet "x86-windows" --x-install-root ".....\buildResources\x86-windows\"

In my x86-windows.cmake file i specified
```
set(VCPKG_TARGET_ARCHITECTURE x86)
set(VCPKG_CRT_LINKAGE static)
set(VCPKG_LIBRARY_LINKAGE static)
```

Am I missing something? Whenever i execute it makes an x64-windows dir in my x86 folder and x64 binaries.

**Comment by Neumann-A** [comment]
> Am I missing something?

Yes, the host triplet which defaults to x64-windows in newer vcpkg-tool. You can modify it by passing `--host-triplet <triplet>`

**Reply by dretax** [reply]
God damn, thanks
